netsniff-ng.c

```diff
@@ -1239,6 +1255,7 @@ static void __noreturn help(void)
 	     "  -R|--rfraw                     Capture or inject raw 802.11 frames\n"
 	     "  -n|--num <0|uint>              Number of packets until exit (def: 0)\n"
 	     "  -P|--prefix <name>             Prefix for pcaps stored in directory\n"
+	     "  -O|--overwrite <N>             File name uses number from 0 to N-1 instead of timestamp. Overwrites file when number wraps around\n"
```

Comment:
The description is a bit too long. Try to shorten it down a bit, e.g.

`O|--overwrite <N>            Limit the number of pcaps to N (file names use numbers 0 to N-1)`

I think the details can be left for the man page.